src/routes/(app)/post/[post_id]/+page.svelte

```diff
@@ -22,26 +22,37 @@
 
     export let data;
     let user_vote: 'UP' | 'DOWN' | null = data.user_vote;
+    let upvotes = data.post.upvotes;
+    let downvotes = data.post.downvotes;
```

Comment:
nit: Could be
```typescript
let { upvotes, downvotes } = data.post;
```
A bit prettier and more cleanly expandable.

Reply:
I'm disappointed in myself as a JS developer.